app/controllers/caffeine/admin/pictures_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Caffeine
+  module Admin
+    class PicturesController < CaffeineController
+
+      def create
+        picture = Caffeine::Picture.create(image: picture_params)
+        render json: {
+          files: [{
+                id: picture.id,
+                url: Refile.attachment_url(picture, :image),
+                delete_url: admin_picture_path(picture.id)
+              }]
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.